core/src/main/java/org/radargun/config/Evaluator.java

```diff
@@ -422,6 +451,15 @@ private Value(String string) {
             } catch (NumberFormatException e2) {
             }
          }
+         if (string.contains(",") && t == ValueType.STRING) {
+            t = ValueType.LIST;
+            String[] rawValues = string.split(",");
+            List<Value> processedValues = new ArrayList<>(rawValues.length);
+            for (String rawValue : rawValues) {
+               processedValues.add(new Value(rawValue.trim()));
```

Comment:
If you eventually find out that you need to use the value as string, the trimming will lose the whitespaces (you can't re-join the elements). Also, single element lists wouldn't be detected.
It seems to me that you should rather add a method that will try to convert the value to list when appropriate.